src/components/RoomsPage/RoomsPage.js

```diff
@@ -29,52 +111,56 @@ const RoomsPage = () => {
           alignment="center"
           margin="0 0 16px 0"
         >
-          <Stack direction="vertical" alignment="left">
-            <Heading as="small">
-              <IconButton className={styles.backButton} margin="0 0 8px 0">
-                <ArrowLeftIcon />
-                All rooms
-              </IconButton>
-            </Heading>
-            <Stack>
-              <GlobePublicIcon className={styles.globePublicIcon} />
-              <Heading as="h2" margin="0 0 0 10px">
-                Bieber's Bungalow
-              </Heading>
-            </Stack>
-          </Stack>
+          <Heading as="h2">Rooms</Heading>
+          <Button size="medium" variant="primary">
+            + Create room
+          </Button>
+        </Stack>
+      </SectionHeader>
+      <Content>
+        <Stack distribution="between" margin="0 0 16px 0">
+          <Heading as="h3">All Rooms</Heading>
           <Stack>
-            <Button size="medium" variant="secondary" margin="0 15px 0 0">
-              Edit settings
-            </Button>
-            <Button size="medium" variant="primary" leftIcon={LinkIcon}>
-              Share
-            </Button>
+            <CircularButton margin="0 16px 0 0">
+              <GridIcon />
+            </CircularButton>
+            <CircularButton active>
+              <ListIcon />
+            </CircularButton>
           </Stack>
         </Stack>
-        <Tabs>
-          <TabItem to={url}>Preview</TabItem>
-          <TabItem to={`${url}/content`}>Content</TabItem>
-          <TabItem to={`${url}/members`}>Members</TabItem>
-          <TabItem to={`${url}/live-insights`}>Live insights</TabItem>
-          <TabItem to={`${url}/moderation`}>Moderation</TabItem>
-        </Tabs>
-      </SectionHeader>
-      <Content className={styles.content}>
-        <Switch>
-          <Route exact path={path} component={PreviewPage} />
-          <Route exact path={`${path}/content`} component={ContentPage} />
-          <Route exact path={`${path}/members`} component={MembersPage} />
-          <Route
-            exact
-            path={`${path}/live-insights`}
-            component={LiveInsightsPage}
-          />
-          <Route exact path={`${path}/moderation`} component={ModerationPage} />
-        </Switch>
+        <Grid cols={4} gap={24}>
```

Comment:
Wondering whether we should instead use 

```suggestion
        <Grid cols="4" gap="24">
```

And perhaps move them towards constants.